src/c++/perf_analyzer/docs/llm.md

```diff
@@ -31,9 +31,55 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 The following guide shows the reader how to use Triton
 [Perf Analyzer](https://github.com/triton-inference-server/client/tree/main/src/c%2B%2B/perf_analyzer)
 to measure and characterize the performance behaviors of Large Language Models
-(LLMs) using Triton with [vLLM](https://github.com/vllm-project/vllm).
+(LLMs) using Triton with [TensorRT-LLM](https://github.com/NVIDIA/TensorRT-LLM) and [vLLM](https://github.com/vllm-project/vllm).
 
-### Setup: Download and configure Triton vLLM Backend
+## Setup: Download and configure Triton Server environment
+
+### Using TensorRT-LLM
+
+1. Follow [step 1](https://github.com/triton-inference-server/tutorials/blob/main/Popular_Models_Guide/Llama2/trtllm_guide.md#installation) of the Installation section.
+
+    This guide will replace the llama model for gpt since it is already included in the tensorrtllm_backend repository release.
```

Comment:
I think it would be better to just stick to llama model since the Katherine's guide uses llama model. Mixing the two might confuse the reader, unless we are going to provide detailed guide for running gpt model.

Reply:
gpt weights also need to be downloaded from huggingface as well (https://github.com/NVIDIA/TensorRT-LLM/tree/main/examples/gpt#1-download-weights-from-huggingface-transformers). But I understand your point since Llama does require extra steps like gaining access to the weights etc.

Reply:
Updated to llama